daffodil-test/src/test/scala/org/apache/daffodil/section07/variables/TestVariables.scala

```diff
@@ -93,6 +93,8 @@ class TestVariables {
 
   @Test def test_unsignedIntVarCast(): Unit = { runner.runOneTest("unsignedIntVarCast") }
 
+  //@Test def test_choiceBranchVariables(): Unit = { runner.runOneTest("choiceBranchVariables") }
```

Comment:
Looks good, but please insert // DAFFODIL-2375 before this line to identify which issue the failing test is for.